409 Conflict  during testCreateDomainNodeIds

Observed in the Windows, Python 3.12, no docker workflow:

```
======================================================================
FAIL: testCreateDomainNodeIds (__main__.DomainTest.testCreateDomainNodeIds)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\hsds\hsds\tests\integ\domain_test.py", line 531, in testCreateDomainNodeIds
    self.assertEqual(rsp.status_code, 201)
AssertionError: 409 != 201
```

--- a/tests/integ/domain_test.py
+++ b/tests/integ/domain_test.py
@@ -521,7 +521,7 @@ class DomainTest(unittest.TestCase):
             self.assertEqual(root_id, rspJson["root"])
 
     def testCreateDomainNodeIds(self):
-        domain = self.base_domain + "/newdomain.h6"
+        domain = self.base_domain + "/newdomain.h7"
         print("testCreateDomainNodeIds", domain)
         headers = helper.getRequestHeaders(domain=domain)
         req = helper.getEndpoint() + "/"

